content/docs/about/kubeflow.md

```diff
@@ -11,6 +11,44 @@ recreate other services, but to provide a straightforward way to deploy
 best-of-breed open-source systems for ML to diverse infrastructures. Anywhere 
 you are running Kubernetes, you should be able to run Kubeflow.
 
+## Getting started with Kubeflow
+
+Follow the [getting-started guide](/docs/started/getting-started) to set up your
+environment.
+
+Then read the [documentation](/docs/) to learn how to use Kubelow.
+Highlights include the following:
+
+* Kubeflow includes services for spawning and managing 
```

Comment:
It might help to explain a little bit why each of these documents is specifically useful for getting started. The descriptions talk about the features, but not what to expect from the document.

Reply:
Instead of elaborating on each bullet point (as the bullets are quite long already) I've changed the intro sentence to indicate that these are specific features of Kubeflow (they're the features that we want to highlight).